target/linux/pistachio/marduk/target.mk

```diff
@@ -14,7 +14,7 @@ DEFAULT_PACKAGES+=kmod-leds-gpio kmod-ledtrig-heartbeat kmod-i2c-core i2c-tools
                   uhttpd uboot-envtools tcpdump board-test \
                   luci kmod-mac802154 \
                   curl libopenssl ca-certificates \
-                  bluez-daemon
+                  bluez-daemon kmod-spi-dev
```

Comment:
Bit random to wack it here when all the kmods are at the top. The lines attempt to group related packages